.gitignore

```diff
@@ -30,3 +30,5 @@ override.tf.json
 
 # Include tfplan files to ignore the plan output of command: terraform plan -out=tfplan
 # example: *tfplan*
+examples/azure-linux-tdx-vm/main.tf
```

Comment:
These should not be here. If they are Git won't track them anymore.

Reply:
Yeah, I noticed that. I didn't put that there- I removed it now.